modules/tensor_mechanics/src/materials/ComputeAxisymmetricRZFiniteStrain.C

```diff
@@ -34,7 +34,7 @@ ComputeAxisymmetricRZFiniteStrain::initialSetup()
 }
 
 Real
-ComputeAxisymmetricRZFiniteStrain::computeDeformGradZZ()
```

Comment:
Never mind about this as well.  After looking at more context, I see what's going on here, and this is fine.